### PR TITLE
Fix docs for advanced audit

### PR DESCRIPTION
**Pull request by CaoShuFeng**
This change updates docs from this pr: #45862

**Release note**:

```
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/apis/audit/types.go [hunk 1]
```diff
@@ -48,7 +48,6 @@ type Event struct {
 	// Time the request reached the apiserver.
 	Timestamp metav1.Time
 	// Unique audit ID, generated for each request.
-	// +optional
```
Comment (CaoShuFeng) [inline staging/src/k8s.io/apiserver/pkg/apis/audit/types.go]: UID is important for identifying a single log. So I think it is not optional.

**Comment by CaoShuFeng** [comment]
@timstclair 
I hope my understanding is right.

**Comment by cblecker** [comment]
@k8s-bot ok to test

**Comment by timstclair** [comment]
Thanks!

/lgtm

File: staging/src/k8s.io/apiserver/pkg/apis/audit/types.go [hunk 2]
```diff
@@ -116,8 +115,8 @@ type PolicyRule struct {
 	// An empty list implies every user.
 	// +optional
 	Users []string
-	// The user groups this rule applies to. If a user is considered matching
-	// if the are a member of any of these groups
+	// The user groups this rule applies to. A user is considered matching
+	// if it has any member of the UserGroups
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/apis/audit/types.go]: if it is a member of any of the UserGroups?

Comment (CaoShuFeng) [inline staging/src/k8s.io/apiserver/pkg/apis/audit/types.go]: Done.

**Comment by timstclair** [comment]
/lgtm

Manually approving docs changes.

**Comment by timstclair** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by CaoShuFeng** [comment]
@k8s-bot gce etcd3 e2e test this

**Pull request merged**